build.gradle

```diff
@@ -60,6 +65,5 @@ eclipse{
 
 task wrapper(type: Wrapper) { gradleVersion = "3.2" }
 
-task stage {
```

Comment:
Needed for https://devcenter.heroku.com/articles/deploying-gradle-apps-on-heroku